pkg/volume/iscsi/iscsi_util.go

```diff
@@ -184,6 +195,17 @@ func (util *ISCSIUtil) DetachDisk(c iscsiDiskUnmounter, mntPath string) error {
 	return nil
 }
 
+func extractTransportname(ifaceOutput string) (iscsiTransport string) {
+	re := regexp.MustCompile(`iface.transport_name = (.*)\n`)
+	iscsiTransport = re.FindStringSubmatch(ifaceOutput)[1]
```

Comment:
I left that out on purpose. As long as `iscsiadm` runs successfully, it will add a line for iface.transport_name on it's own even if none is specified. It is pretty much impossible for the regex to fail.